modules/react/common/stories/Common.mdx

```diff
@@ -0,0 +1,103 @@
+import {ExampleCodeBlock} from '@workday/canvas-kit-docs';
+
+import {BaseExample} from './mdx/examples/BaseExample';
+
+
+<Meta title="Hooks and Utilities/Common" />
+
+# Common
+The Common package contains a collection of components and utilities that serve as the foundation for building robust features across other packages.
+
+## Installation
+
+```sh
+yarn add @workday/canvas-kit-react
+```
+
+## Base
+
+The `Base` component is a low level component that automatically handles the [`cs`](https://workday.github.io/canvas-kit/?path=/docs/styling-basics--docs#cs-prop) prop for you and style merging via [`handleCsProp`](https://workday.github.io/canvas-kit/?path=/docs/styling-merging-styles--docs#handlecsprop).
+As we transition away from runtime styles, you should use `Base` when styling an element where you want more control over the styles and a Canvas Kit component doesn't quite fit your needs.
+
+### Usage
+
+Use `Base` when you need to statically style a container element or you're building a complex component that is **not reusable**.
```

Comment:
Considering if there's a better way to craft this paragraph. I understand the use case of:

> I want to statically style this basic HTML element (`p`, `div`, etc).

The part about statically styling a container element gets at this.

Can you elaborate on why you would only use `Base` to build a complex component that is **not** reusable? I ask since I could imagine someone wanting to compose a complex reusable component out of `Base` components.